lib/index.js

```diff
@@ -270,7 +270,8 @@ module.exports = class DockerComposeParser {
       metadata: service.metadata,
       contextVersion: Promise.props({
         advanced: true,
-        buildDockerfilePath: Parser.buildDockerfilePathParser({ build: rawService.build, warnings })
+        buildDockerfilePath: Parser.buildDockerfilePathParser({ build: rawService.build, warnings }),
+        code: Parser.buildRemoveCodeParser({ build: rawService.build, warnings })
```

Comment:
Do we actually plan on adding this property to the contextVersion schema and saving this? I think this is 100% cool if that's what we're doing, but wouldn't want to pollute the object if that's not the plan.